helm.tf

```diff
@@ -99,191 +82,830 @@ resource "helm_release" "metrics_server" {
+    value = var.loki_gateway_max_replicas
+  }  
+
+  dynamic "set" {
+    for_each = var.loki_gateway_requests_cpu != null ? ["do it"] : []
+    content {
+      name  = "gateway.resources.requests.cpu"
+      value = var.loki_gateway_requests_cpu
+    }
   }
 
-# ========================= prometheus ========================= #
+  dynamic "set" {
+    for_each = var.loki_gateway_requests_ram != null ? ["do it"] : []
+    content {
+      name  = "gateway.resources.requests.memory"
+      value = var.loki_gateway_requests_ram
+    }
+  }
+
+  dynamic "set" {
+    for_each = var.loki_gateway_limits_cpu != null ? ["do it"] : []
+    content {
+      name  = "gateway.resources.limits.cpu"
+      value = var.loki_gateway_limits_cpu
+    }
+  }
+
+  dynamic "set" {
+    for_each = var.loki_gateway_limits_ram != null ? ["do it"] : []
+    content {
+      name  = "gateway.resources.limits.memory"
+      value = var.loki_gateway_limits_ram
+    }
+  }
+
+  set {
+    name  = "gateway.ingress.enabled"
+    value = var.loki_gateway_ingress_enabled
+  }  
+
   set {
-    name  = "prometheus.enabled"
-    value = var.prometheus_enabled
+    name  = "gateway.ingress.hosts[0].host"
+    value = var.loki_gateway_ingress_host
   }
 
-# prometheus - alertmanager persistence
   set {
-    name  = "prometheus.alertmanager.persistentVolume.enabled"
-    value = var.prometheus_alertmanager_persistence_enabled
+    name  = "gateway.ingress.ingressClassName"
+    value = var.loki_gateway_ingress_class_name
   }
 
   set {
-    name  = "prometheus.alertmanager.persistentVolume.storageClassName"
-    value = var.prometheus_alertmanager_persistence_storage_class_name
+    name  = "gateway.ingress.hosts[0].paths[0].path"
+    value = var.loki_gateway_ingress_path
   }
 
   set {
-    name  = "prometheus.alertmanager.persistentVolume.size"
-    value = var.prometheus_alertmanager_persistence_size
+    name  = "gateway.ingress.hosts[0].paths[0].pathType"
+    value = var.loki_gateway_ingress_path_type
   }
 
 
-# prometheus - server
+}
+
+# ================== fluent-bit ================== #
+
+resource "helm_release" "fluent_bit" {
+  count             = var.helm_fluent_bit_enabled ? 1 : 0
+  name              = "fluent-bit"
+  namespace         = "monitoring"
+  create_namespace  = true
+  repository        = "https://fluent.github.io/helm-charts"
+  chart             = "fluent-bit"
+  version           = "0.19.24"
+  dependency_update = true
+
+  values            = [
+    file("${path.module}/helm-values/fluent-bit.yaml")
+  ]
+
+
+}
+
+# ================== tempo-distributed ================== #
+
+resource "helm_release" "tempo_distributed" {
+  count             = var.helm_tempo_enabled ? 1 : 0
+  name              = "tempo-distributed"
+  namespace         = "monitoring"
+  create_namespace  = true
+  repository        = "https://grafana.github.io/helm-charts"
+  chart             = "tempo-distributed"
+  version           = "0.17.1"
```

Comment:
Allow version selection (module input)

Reply:
I'm marking this as resolved because we agreed to don't do it